sample.env

```diff
@@ -23,6 +23,7 @@ REACT_APP_WALLET_MIGRATION_URL=https://wallets.shapeshift.com/api/migrate
 REACT_APP_FRIENDLY_CAPTCHA_SITE_KEY=FCMM7AFC0S6A8NUK
 
 REACT_APP_FEATURE_OSMOSIS=false
+REACT_APP_FEATURE_COINBASE_RAMP=false
```

Comment:
This should also be added to `src/state/slices/preferencesSlice/preferencesSlice.ts`:
https://github.com/shapeshift/web/blob/25482f7ac43b041326887896fe13afadd179ed7e/src/state/slices/preferencesSlice/preferencesSlice.ts#L6-L8
https://github.com/shapeshift/web/blob/25482f7ac43b041326887896fe13afadd179ed7e/src/state/slices/preferencesSlice/preferencesSlice.ts#L20-L22

Or else it won't show up in the flags UI:
<img width="1261" alt="image" src="https://user-images.githubusercontent.com/17035424/173462478-c2212906-3a51-4337-8d62-31390eae867e.png">

Reply:
Good find. Done.